src/components/ui/Breadcrumb/breadcrumb.module.scss

```
@@ -0,0 +1,165 @@
+@import "src/styles/scaffold";
+
+.fs-breadcrumb {
+  // --------------------------------------------------------
+  // Design Tokens for Breadcrumb
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-breadcrumb-margin-left                         : calc(-1 * var(--fs-spacing-1));
+
+  // List item
+  --fs-breadcrumb-list-item-padding                   : var(--fs-spacing-0);
+
+  --fs-breadcrumb-list-item-max-width-mobile          : 30%;
+
+  // Link
+  --fs-breadcrumb-link-color-visited                  : var(--fs-color-link);
+
+  // Link First
+  --fs-breadcrumb-link-first-padding                  : var(--fs-spacing-1);
+  --fs-breadcrumb-link-first-border-radius            : var(--fs-border-radius-circle);
+  --fs-breadcrumb-link-first-hover-background-color   : var(--fs-color-primary-bkg-light);
+
+  // Icon
+  --fs-breadcrumb-item-icon-color                     : var(--fs-color-text);
+
+  // Divider
+  --fs-breadcrumb-divider-height                      : var(--fs-spacing-3);
+  --fs-breadcrumb-divider-margin                      : var(--fs-spacing-1);
+  --fs-breadcrumb-divider-border-left-width           : var(--fs-border-width);
+  --fs-breadcrumb-divider-border-left-color           : var(--fs-border-color-light);
+
+  // Dropdown Button
+  --fs-breadcrumb-dropdown-button-padding             : 0 calc(var(--fs-spacing-0) / 2);
+  --fs-breadcrumb-dropdown-button-margin              : calc(var(--fs-spacing-0) / 4);
+  --fs-breadcrumb-dropdown-button-color               : var(--fs-color-link);
+  --fs-breadcrumb-dropdown-button-border-radius       : var(--fs-spacing-0);
+  --fs-breadcrumb-dropdown-button-transition-property : var(--fs-transition-property);
+  --fs-breadcrumb-dropdown-button-transition-timing   : var(--fs-transition-timing);
+  --fs-breadcrumb-dropdown-button-transition-function : var(--fs-transition-function);
+
+  // Dropdown Button Text
+  --fs-breadcrumb-dropdown-button-text-size           : var(--fs-text-size-3);
+  --fs-breadcrumb-dropdown-button-text-weight         : var(--fs-text-weight-bold);
+  --fs-breadcrumb-dropdown-button-text-transform      : translate(rem(1px), rem(-6px));
```

I think we can't use the rem() method in CSS variables definition. We are converting manually in other locals:
```suggestion
  --fs-breadcrumb-dropdown-button-text-transform      : translate(0.063rem, -0.375rem)
```

---

This rule seems to be irrelevant to the component, could you check that?

---

That's true! It seems irrelevant. I removed [here](https://github.com/vtex-sites/nextjs.store/pull/162/commits/e5d65ed373c5ae357861c07b2ec146dd0948b968).